internal/performance/export_test.go

```diff
@@ -55,12 +53,19 @@ func TestExport(t *testing.T) {
 		criteria = publishdb.IterateExposuresCriteria{
 			OnlyLocalProvenance: false,
 		}
-		want           = keysPerPublish * *numPublishes
+		numPublishes   = 100000
+		want           = keysPerPublish * numPublishes
 		batchStartTime = time.Now().Add(time.Duration(-totalBatches-10) * exportPeriod)
-		roughPerBatch  = *numPublishes/totalBatches + 1
+		roughPerBatch  = numPublishes/totalBatches + 1
 	)
 
-	makoQuickstore, cancel := setup(t, *makoDev)
+	if p, err := strconv.Atoi(os.Getenv("PUBLISHES")); err == nil {
```

Comment:
I'm a bit hesitate to allow `PUBLISHES` to be overrode in `prod` env, can we somehow couple this with `dev` env so that we can better prevent unexpected data from being pushed to prod benchmark.

Reply:
I see, I had placed it within the `dev` condition of the `.sh` script so that it could not be set without `--dev` from the command. But someone could still manually set it. I can also check for `PERFORMACE_DEV` 👍